platform-common/src/main/java/com/softicar/platform/common/core/exception/ExceptionsCollector.java

```diff
@@ -98,4 +105,31 @@ public void consumeIfNotEmpty(Consumer<String> messageConsumer) {
 			messageConsumer.accept(getMessage());
 		}
 	}
+
+	public static class ExceptionCollection {
+
+		private final List<StackTraceElement> stackTraceElements;
+		private final List<Throwable> throwables;
+
+		public ExceptionCollection(List<StackTraceElement> stackTraceElements) {
+
+			this.stackTraceElements = stackTraceElements;
+			this.throwables = new ArrayList<>();
+		}
+
+		public List<StackTraceElement> getStackTraceElements() {
+
+			return Collections.unmodifiableList(stackTraceElements);
+		}
+
+		public List<Throwable> getThrowables() {
```

Comment:
Calling `getThrowables()` on an `ExceptionCollection` instance seems particularly odd to me. Would bother me less if the method was `public List<Throwable> getExceptions()` or if the class would have `Throwable` in its name.